.ci/run.sh

```diff
@@ -73,7 +73,20 @@ case "${CI_JOB}" in
 	"METRICS")
 		export RUNTIME="kata-runtime"
 		export CTR_RUNTIME="io.containerd.run.kata.v2"
-		echo "INFO: Running metrics tests"
+		echo "INFO: Running qemu metrics tests"
+		sudo -E PATH="$PATH" ".ci/run_metrics_PR_ci.sh"
```

Comment:
hmm.. can you create a symlink to `configuration-qemu.toml` before running qemu ? - just to make sure the test is using the right configuration file